docs/content/Deep-Storage.md

```diff
@@ -54,3 +54,17 @@ If you are using the Hadoop indexer, set your output directory to be a location
 
 Please note that this is a community contributed module and does not support Cassandra 2.x or hadoop-based batch indexing. For more information on using Cassandra as deep storage, see [Cassandra Deep Storage](Cassandra-Deep-Storage.html).
 
+## Azure
+
+[Microsoft Azure Storage](http://azure.microsoft.com/en-us/services/storage/) is another option for deep storage. This requires some additional druid configuration.
+
+```
+druid.storage.type=azure
```

Comment:
General note: The whole Deep Storage doc could use a reformat to use the property table stuff seen in other parts of the doc (not required for this PR)